content/docs/extensions/timescaledb.md

```diff
@@ -0,0 +1,235 @@
+</Admonition>
+
+**Version Availability:**
+
+The version of `timescaledb` available on Neon varies based on which version of Postgres you select for your project.
+
+- Postgres 14 - `timescaledb` 2.10.1
+- Postgres 15 - `timescaledb` 2.10.1
+- Postgres 16 - `timescaledb` 2.13.0
+
+_Only Apache-2 licensed features are supported. Compression is not supported._
+
+## Enable the `timescaledb` extension
+
+You can enable the extension by running the following `CREATE EXTENSION` statement in the Neon **SQL Editor** or from a client such as `psql` that is connected to Neon.
+
+```sql
+CREATE EXTENSION IF NOT EXISTS timescaledb;
+```
+
+For information about using the Neon SQL Editor, see [Query with Neon's SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor). For information about using the `psql` client with Neon, see [Connect with psql](/docs/connect/query-with-psql-editor).
+
+## Basic functions for working with Hypertables and chunks
+
+Create a `hypertable` for temperature data:
+
+```sql
+CREATE TABLE temperature_data (
+ time       TIMESTAMPTZ NOT NULL,
+ location   TEXT        NOT NULL,
+ temperature DOUBLE PRECISION
+);
+```
+
+Create a `hypertable` for temperature data using `create_hypertable`:
+
+
+```sql
+SELECT create_hypertable('temperature_data', 'time');
+```
+
+Insert data in `temperature_data` table:
+
+```sql
+INSERT INTO temperature_data(time, location, temperature)
+SELECT
+ generate_series('2023-01-01'::TIMESTAMPTZ, '2023-01-05'::TIMESTAMPTZ, '1 hour'::INTERVAL),
+ 'Room A',
+ random() * 10 + 20;
+
+
+INSERT INTO temperature_data(time, location, temperature)
+SELECT
+ generate_series('2023-01-01'::TIMESTAMPTZ, '2023-01-05'::TIMESTAMPTZ, '1 hour'::INTERVAL),
+ 'Room B',
+ random() * 10 + 22;
+```
+
+Show chunks information:
+
+
+```sql
+SELECT show_chunks('temperature_data');
+```
+
+Returns:
+
+```text
+|             show_chunks              |
+|--------------------------------------|
+| _db_internal._hyper_1_1_chunk|
+| _db_internal._hyper_1_2_chunk|
+```
+
+Show detailed chunks information:
+
+
+```sql
+SELECT * FROM chunks_detailed_size('temperature_data')
+ ORDER BY chunk_name;
+```
+
+Returns:
+
+```text
+|     chunk_schema     |    chunk_name    | table_bytes | index_bytes | toast_bytes | total_bytes | node_name |
+|----------------------|------------------|-------------|-------------|-------------|-------------|-----------|
+| _db_internal | _hyper_1_1_chunk |       40960 |       16384 |        8192 |       65536 |           |
+| _db_internal | _hyper_1_2_chunk |        8192 |       16384 |        8192 |       32768 |           |
+```
+
+Add `location` as a dimension:
+
+
+```sql
+SELECT add_dimension('temperature_data', 'location', number_partitions => 2);
+```
+
+Returns:
+
+```text
+|             add_dimension            |
+----------------------------------------
+|(4,public,temperature_data,location,t)|
+```
+
+## Use Hyperfunctions to analyze data
+
```

Comment:
Add some basic context:

"Now that your table contains data, you can perform various queries and functions to analyze it."

The DO article makes it clear that you can work with both SQL and timescaledb functions to analyze data, and starts by showing several SQL commands:

"As mentioned before, to work with hypertables you can use standard SQL commands. For example, to show the last 10 entries from the conditions hypertable, run the following command:"